lib/fastlane/actions/unlock_keychain.rb

```diff
@@ -4,20 +4,28 @@ class UnlockKeychainAction < Action
       def self.run(params)
         keychain_path = self.expand_keychain_path(params[:path])
         add_to_search_list = params[:add_to_search_list]
+        set_default = params[:set_default]
+        commands = []
 
         if keychain_path.empty?
           raise "Could not find the keychain file: #{keychain_path}".red
         end
 
         # add to search list if not already added
-        if add_to_search_list
-          add_keychain_to_search_list(keychain_path)
+        if add_to_search_list == true || add_to_search_list == :add
```

Comment:
You can use `:add` or `:replace` as the `:add_to_search_list:` value. I forget to add documentation, I'll add this missing information.